template.yaml

```diff
@@ -131,15 +131,6 @@ Resources:
     Properties:
       EmailIdentity: !Sub 'scopus@${CustomDomain}'
 
-  AllowSiktEmailsToBeReceived:
```

Comment:
Is this referenced anywhere?

Reply:
In SES this would have placed an ip-filter for incomming emails on all SES emails in that account. Sadly microsoft is on azure serverless so the ip's changes constantly and there is no sensible range I can see.
And by allowing all azure ip's have we really blocked anything?